pkg/controller/controller.go

```diff
@@ -102,6 +102,13 @@ func (c *modifyController) Run(workers int, ctx context.Context) {
 		return
 	}
 
+	// On startup, queue all existing PVCs - this ensures that PVCs
+	// awaiting modification (or otherwise in an inconsistent state)
+	// are processed immediately instead of waiting on a resync
+	for _, claim := range c.claims.List() {
```

Comment:
What are the scalability implications of this for a cluster with say 50k claims?  

I don't see the resizer controller doing this. They just wait for cache sync on controller startup like we do? 

https://github.com/kubernetes-csi/external-resizer/blob/ef3b511a3a2b6c888a585763093c57b20a646b53/pkg/controller/controller.go#L283

Reply:
> What are the scalability implications of this for a cluster with say 50k claims?

Essentially zero as this operation is free for all claims that don't need to be synced. If the leader election was constantly flapping between replicas it could cost some API calls but any scalability on such a broken cluster would be entirely shot anyways.

> I don't see the resizer controller doing this.

This isn't the resizer controller. I can't speak to how they solve this problem, I don't know their codebase well enough.